src/globals.js

```diff
@@ -23,10 +26,15 @@ httpsAgent.createConnection = (port, host, options) => {
   return socket;
 };
 
-module.exports = {
+function resetColdstart(bool = false) {
```

Comment:
I like this as a function, but `resetColdstart` makes me think it resets the coldstart value ... which would set it to true, not false (because it starts true). 🤔  pondering other name suggestions